prow/cmd/phaino/local.go

```diff
@@ -85,6 +86,12 @@ func readRepo(ctx context.Context, path string) (string, error) {
 		return "", fmt.Errorf("workingDir: %v", err)
 	}
 	def, err := findRepo(wd, path)
+	if err != nil { // If k8s/test-infra is not under GOPATH, find under GOPATH.
+		pkg, err := build.Default.Import(path, build.Default.GOPATH, build.FindOnly|build.IgnoreVendor)
```

Comment:
this err shadows the err above, given the block below I think you want something like:
```suggestion
		pkg, err2 := build.Default.Import(path, build.Default.GOPATH, build.FindOnly|build.IgnoreVendor)
		err = err2
```
https://play.golang.org/p/qzBp525zKbW

Reply:
should add: I think the more idiomatic approach is to streamline this (perhaps breaking into some helpers), but this is at least functionally correct